bindings/python/README.rst

```diff
@@ -160,6 +160,11 @@ binaries, and extract it::
   $ curl -O https://s3.amazonaws.com/mciuploads/libmongocrypt/all/master/$LIBMONGOCRYPT_REVISION/libmongocrypt-all.tar.gz
   $ mkdir libmongocrypt-all && tar xzf libmongocrypt-all.tar.gz -C libmongocrypt-all
 
+.. note:: If you are installing PyMongoCrypt from a source tarball, use the following command
```

Comment:
Let's not use a `.. note` since it doesn't render nicely on github.

Reply:
Done.